SU-HSDO/suhumsci
# Hsd8-384 Set prefix on config entities

# READY FOR REVIEW

# Summary
- Force all new custom config entities to be prefixed with `custm` on production site.

# Needed By (Date)
- 10/11

# Urgency
- High

# Steps to Test
1. checkout this branch
1. sync archaeology `blt drupal:sync --site=archaeology`
1. in the UI create any new config entity (content type, taxonomy vocab, view, image style, etc)
1. ensure no matter what you enter for the machine name, it will be prefixed with `hs_`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
